redash/handlers/queries.py

```diff
@@ -488,9 +488,9 @@ def post(self, query_id):
 
         parameter_values = collect_parameters_from_request(request.args)
         parameterized_query = ParameterizedQuery(query.query_text, org=self.current_org)
-
+        apply_auto_limit = "apply_auto_limit" in query.options and query.options["apply_auto_limit"]
```

Comment:
This can be simplified (and be more idiomatic) using the `get` method of `dict`.

```suggestion
        apply_auto_limit = query.options.get("apply_auto_limit", False)
```